HNG_model.py

```diff
@@ -46,16 +49,19 @@
 print(sol_vec)
 
 #TODO #2
-A_spec = (10*initial['r_0'])**2/(V_elect) #Specific surface of reaction (m²/m³) using r_0 for scale
+A_spec = (10*initial['r_0'])**2/(V_elect)
+#Reaction surface area/volume of electrolyte, used if the rate of reactions is mol/m², I think used in nucleation, Specific surface of reaction (m²/m³) using r_0 for scale
 
 #TODO #3
 int_volume =  2/3*initial['r_0']**3
+#initiual volume of a nucleation
+
 nucleii = 10
 print (A_spec)
```

Comment:
Not a big deal, but I'm guessing this `print` statement was for debugging purposes.  Eventually would be nice to clean it up.